go_modules/helpers/go.sum

```diff
@@ -4,3 +4,22 @@ github.com/dependabot/gomodules-extracted v0.0.0-20181020215834-1b2f850478a3 h1:
 github.com/dependabot/gomodules-extracted v0.0.0-20181020215834-1b2f850478a3/go.mod h1:+dRXSrUymjpT4yzKtn1QmeknT1S/yAHRr35en18dHp8=
 github.com/dependabot/gomodules-extracted v1.1.0 h1:k0Fcin3JZBfd/yW0JEeg38UtTiZl6E9uYkdqA4Dqbyw=
 github.com/dependabot/gomodules-extracted v1.1.0/go.mod h1:+dRXSrUymjpT4yzKtn1QmeknT1S/yAHRr35en18dHp8=
```

Comment:
Would you mind running `go mod tidy` in this dir?

Reply:
done!